scripts/deploy.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+echo "$HEROKU_API_KEY" | docker login -u "$HEROKU_LOGIN" --password-stdin registry.heroku.com
+docker build --target production --tag olutiab/olutiab-todo-app .
+docker tag olutiab/olutiab-todo-app registry.heroku.com/olutiab-todo-app/web
+docker push olutiab/olutiab-todo-app
+docker push registry.heroku.com/olutiab-todo-app/web
+heroku container:release web --app olutiab-todo-app
```

Comment:
Minor, but you have set a HEROKU_APP key via the Travis UI that you don't use here; that's fine, but means that this actually gets hidden in the travis logs even though the value isn't really hidden. I'd recommend removing the HEROKU_APP key or adding it as a public-facing id, but otherwise adjusting these commands to use that would also be reasonable